index.html

```diff
@@ -56,5 +56,124 @@ <h1>Hey there.<br />I’m Victor.<br /></h1>
+              <li>HTML</li>
+            </ul>
+            <button class="project-btn btn">See project</button>
+          </div>
+        </article>
+        <article class="project-cards flex-column">
+          <div class="content-cards flex-column">
+            <h3>Multi-Post Stories<br />Gain+Glory</h3>
+            <ul class="technologies flex-row">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button class="project-btn btn">See project</button>
+          </div>
+        </article>
+        <article class="project-cards flex-column">
+          <div class="content-cards flex-column">
+            <h3>Multi-Post Stories<br />Gain+Glory</h3>
+            <ul class="technologies flex-row">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button class="project-btn btn">See project</button>
+          </div>
+        </article>
+        <article class="project-cards flex-column">
+          <div class="content-cards flex-column">
+            <h3>Multi-Post Stories<br />Gain+Glory</h3>
+            <ul class="technologies flex-row">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button class="project-btn btn">See project</button>
+          </div>
+        </article>
+      </section>
+      <section class="about-section">
+        <div class="about-me flex-column">
+          <h2>About me</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <button class="resume-btn btn">Get my resume</button>
+        </div>
+        <div class="skillcards-container flex-column">
+          <article class="card-1 skill-cards flex-column">
+            <h3>Languages</h3>
+            <ul class="tags flex-row">
+              <li>JavaScript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          </article>
+          <article class="card-2 skill-cards flex-column">
+            <h3>Frameworks</h3>
+            <ul class="tags flex-row">
+              <li>Reac.js</li>
+              <li>Ruby on Rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+              <li>Selenium</li>
+            </ul>
+          </article>
+          <article class="card-3 skill-cards flex-column">
+            <h3>Skills</h3>
+            <ul class="tags flex-row">
+              <li>Database Management</li>
+              <li>Version Control</li>
+              <li>CLI</li>
+              <li>Web development</li>
```

Comment:
- [ ] Nice work on the "about me" section and on the whole project in general it looks identical to the Figma design, however, from a semantic side what would be more appropriate to use here instead of `<article>` tags are `<li>` tags which was required by the project's [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) 🔽  please work on that

  > Please use `ul` and `li` for categories in "About myself" section. `Languages`, `Frameworks`, and `Skills` should be list items.